<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>77</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="layoutPlot"/>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="3">
      <widget class="QPushButton" name="btnSelectSamples">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="3">
      <widget class="QLineEdit" name="leMapAgreement"/>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QSpinBox" name="spnNeigbourhood">
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="btnSelectReport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>R-squared</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Kappa</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Map agreement</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <widget class="QPushButton" name="btnTrainNetwork">
       <property name="text">
        <string>Train neural network</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="3">
      <widget class="QLineEdit" name="leKappa"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Maximum iterations</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QSpinBox" name="spnMaxIterations">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="3">
      <widget class="QLineEdit" name="leRSquared"/>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QDoubleSpinBox" name="spnLearnRate">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QLineEdit" name="leSamplesPath">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Neighbourhood</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="chkSaveSamples">
       <property name="text">
        <string>Save samples</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Learning rate</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="chkCreateReport">
       <property name="text">
        <string>Create report</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QLineEdit" name="leReportPath">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="leTopology"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Network topology</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Î” RMS</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="leDeltaRMS">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Momentum</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QDoubleSpinBox" name="spnMomentum">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Validation RMS error</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QLineEdit" name="leValidationError">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>verticalSpacer</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
